packages/dcos-integration-test/extra/conftest.py

```diff
@@ -34,13 +34,11 @@ def pytest_collection_modifyitems(session, config, items):
 @pytest.fixture
 def vip_apps(dcos_api_session):
     vip1 = '6.6.6.1:6661'
-    test_app1, _ = get_test_app()
-    test_app1['portDefinitions'][0]['labels'] = {
-        'VIP_0': vip1}
-    test_app2, _ = get_test_app()
-    test_app2['portDefinitions'][0]['labels'] = {
-        'VIP_0': 'foobarbaz:5432'}
-    vip2 = 'foobarbaz.marathon.l4lb.thisdcos.directory:5432'
+    test_app1, _ = get_test_app(vip=vip1)
+    name = 'myvipapp'
+    port = 5432
+    test_app2, _ = get_test_app(vip='{}:{}'.format(name, port))
```

Comment:
I did not conveniently skip that line. The word is "prefer", not "must". The point of using the word "prefer" is that I think we can all agree that splitting hairs and arguing over python string formatting is a giant waste of everyone's time given the variety of python string formatting options available.

This is not a matter of practice. Its just common sense to make things readable. This string only serves to add a SINGLE COLON and you want to have at least 4 variable references and a function call to complete it. If anything, I would rather do:
```host + ':' + port```
Anyone off the street could make sense of that^^
but you think THIS is better?
```'{host}:{port}'.format(host=host, port=port)```
Now this is hopelessly python specific and will appear like chinese to anyone who is not familiar with wacky python paradigms
My point is, these kind of things take nontrival effort (e.g. deciding what silly name to use or making commits and having to rerun tests) and add almost no value whatsoever except for conforming only for the sake of conforming.

Let's try to stick to making this criticism when its more appropriate.